sql/2021/http/pages_most_requests.sql

```diff
@@ -0,0 +1,18 @@
+# standardSQL
+# Pages loading most number of requests
```

Comment:
Nit: This looks like a repeat from previous years, but I think it'd be better as a distribution with 100th percentile for "most requests" but also a sense of the rest of the curve.

Reply:
Actually this is a new one for this year! It's looking for extreme stats for one of those call outs of a memorable stat: "The most number of resources loaded is 17,923 requests!" I gave a few examples (top 100) so we could confirm the sites do use that many and not a temporarily crawling error.